services/src/northbound-service/northbound/src/test/resources/northbound.properties

```diff
@@ -12,6 +12,8 @@ kafka.hosts=kafka.pendev:9092
 
 server.contextPath=/api
 
+web.request.asyncTimeout=30000
```

Comment:
Can we have "default" value for this option? So its presence in property file will become optional.

Reply:
why not to follow pattern that is used here?